Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -2722,15 +2763,15 @@ script:
     - defaultValue: '0'
       description: The UTC timestamp in seconds of the last update. The incident or detection is only updated if it was modified after the last update time.
       name: lastUpdate
-    description: Gets remote data from a remote incident or detection. This method does not update the current incident or detection, and should be used for debugging purposes only.
+    description: Gets remote data from a remote incident or detection. This method does not update the current incident or detection, and should be used for debugging purposes only. Note that this command is supported in Cortex XSOAR only.
```

Comment:
can we hide these commands in XSIAM?

Reply:
it can be done by setting `hidden:marketplacev2: true`. However, this is not supported in the content item schema in the SDK so a validation error is raised. We can discuss offline if you would like more details.